dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1950,6 +1950,11 @@ $modal-no-icon-gap: 22px;
   }
 }
 
+#feedbackBlocksContainer.hidden {
```

Comment:
Know what, it might be worth naming this something other than `hidden`, even though it's scoped, just in case something gets called `.hidden` on a level page down the road that adds another property / hides in an unexpected way.